### ISSUE TITLE
Go button should select the top word in the list

### ISSUE THREAD

**Issue by interestinall** [wordset/wordset#121]
I am speculating that this is because "breakable" and other variants of break are listed prior but am not sure.

![img_0012](https://cloud.githubusercontent.com/assets/4720348/10148865/9fd305c6-65fb-11e5-8684-42970b1fc40a.PNG)

**Comment by HamptonMakes** [comment]
I'd rephrase this as "Go button should select the top word in the list"

**Comment by interestinall** [comment]
works for me

**Comment by HamptonMakes** [comment]
Duplicate of #115
